### DISCUSSION TITLE
Kernel Source 7.56 -> 7.57 nicht verändert

### DISCUSSION THREAD

**Discussion by hoinzy** [Freetz-NG/freetz-ng#972] [Allgemein] [answered]
Laut AVM wurden die Open-Source komponenten von 7.56 auf 7.57 bei der FritzBox 6850 5G verändert. Ich sehe bei make menuconfig jedoch mit der aktuellen 7.57 keinen Eintrag zum ersetzen des Kernels. Ich nehme an das Script sucht nur nach Dateien mit der gleichen Versionsnummer.

Wie kann ich die 7.56 Kernel Sources für 7.57 verwenden?

**Comment by fda77** [comment, marked as answer]
http://osp.avm.de/fritzbox/fritzbox-6850-5g/
https://freetz-ng.github.io/freetz-ng/SUPPORT.html#quellcode
Wenn dann BEIDE Dateien GLEICHZEITIG verfügbar sind kannst du eine diff davon machen

Für das was Freetz damit macht sollte es meist egal sein, AVM rückt eh nicht zu 100% das heraus was zum Imagebau genutzt wurde, siehe
https://github.com/search?q=repo%3AFreetz-NG%2Ffreetz-ng+crappy&type=commits

Replace kernel gibt es für kernel ab v4 eh nicht mehr, weil es niemand gemach hat (du kannst der erst sein!)
